resources/views/projects/details.blade.php

```diff
@@ -80,67 +83,124 @@
 					@foreach($project->labels as $label)
-						<div class="ui label pill">{{ $label->name }}</div>
+					<div class="ui label pill">{{ $label->name }}</div>
 					@endforeach
 				</div>
 				<!-- Skillset -->
 				<div class="ui detail-container">
 					<p><strong>Required skillset</strong></p>
 					@foreach($project->skills as $skill)
-						<div class="ui label pill">{{ $skill->name }}</div>
+					<div class="ui label pill">{{ $skill->name }}</div>
+					@endforeach
+				</div>
+			</div>
+			<!-- Progress section -->
+			<div class="sixteen wide column">
+				<h3>Progress</h3>
+				<div id="progress" class="ui blue progress" data-percent="{{ $project->progress * 100 }}">
+					<div class="bar">
+						<div class="progress">{{ $project->progress * 100 }}%</div>
+					</div>
+					<div class="label">Currently on: {{ $project->milestones->where('status', Config::get('enum.milestone_status')['current'])->first()->name }}</div>
+				</div>
+				@if(isset($project->milestones))
+				<div class="ui list milestone-section">
+					@foreach ($project->milestones as $milestone)
+
+					@switch($milestone->status)
+					@case(Config::get('enum.milestone_status')['done'])
+					<div class="item done">
+						<i class="large circle green icon"></i>
+						<div class="content">
+							<p class="header">{{ $milestone->name }}</p>
+							<div class="description">Done on {{ $milestone->done_date}}</div>
+						</div>
+					</div>
+					@break
+
+					@case(Config::get('enum.milestone_status')['current'])
+					<div class="item current">
+						<i class="large circle blue icon"></i>
+						<div class="content">
+							<p class="header">{{ $milestone->name }}</p>
+							<div class="description"></div>
+						</div>
+					</div>
+					@break
+
+					@default
+					<div class="item coming-up">
+						<i class="large circle grey icon"></i>
+						<div class="content">
+							<p class="header">{{ $milestone->name }}</p>
+							<div class="description"></div>
+						</div>
+					</div>
+					@endswitch
 					@endforeach
+					@endif
 				</div>
 			</div>
 		</div>
 	</div>
+
 	<div class="ui bottom attached tab segment" data-tab="collaborators">
 		<div class="ui stackable grid">
 			@if(isset($project->team->members) && count($project->team->members) > 0)
-				<div class="eight wide column">
-					<h2>Owners</h2>
-					<table class="ui striped table">
-						<tbody>
-							@foreach($project->team->members as $member)
-								<tr class="selectable">
-									<td>
-										<a href="{{ url('users', $member->id) }}">
-											<img class="ui mini circular image" src="{{ isset($member->picture_url) ? $member->picture_url : 'https://dummyimage.com/400x400/3498db/ffffff.png&text=B' }}" alt="{{ $member->name }}" style="display: inline; margin-right: 10px;"/>
-											{{ $member->name }}
-										</a>
-									</td>
-								</tr>
-							@endforeach
-						</tbody>
-					</table>
-				</div>
+			<div class="eight wide column">
```

Comment:
Bueno ya supongo que en el diff se ve donde borraste la indentaci&oacute;n. Not sure why but please fix this :)

Reply:
Creo que VSCode no es bueno con su comando de Reindent lines 😬, ahorita lo pongo bien!